### PR TITLE
feat: Update cozy-ui from 48.0.0 to 93.1.1

### PR DESCRIPTION
**Pull request by cballevre**
This migration is necessary as part of the version upgrade of cozy-ui on cozy-drive to keep the DevTools compatible

### CODE REVIEW

File: packages/cozy-client/package.json
Comment (Crash--) [inline packages/cozy-client/package.json:1]: You should not put a space after CHANGE. I think it'll not work.

Comment (Crash--) [reply packages/cozy-client/package.json:1]: Is it needed only if the app use CozyDevTools? If yes, maybe we should add that to the commit message

File: packages/cozy-client/package.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
     "cozy-flags": "2.10.2",
     "cozy-logger": "1.7.0",
     "cozy-ui": "93.1.1",
+    "cozy-intent": "1.17.3",
```
Comment (Crash--) [inline packages/cozy-client/package.json:55]: same here for the commit message

**Review by Crash-- — APPROVED** [review]
small things to change, but LGTM. Thanks.

File: packages/cozy-client/package.json
Comment (cballevre) [reply packages/cozy-client/package.json:1]: Thx I did it the french way 🙁

**Pull request merged**